setup.py

```diff
@@ -135,6 +135,7 @@ def run(self):
         'jsonpath_rw',
         'requests',
         'requests_gssapi',
+        'ruamel.yaml',
```

Comment:
I'm checking to see if this is available on el7 + el8 + el9

Reply:
ruamel has been dropped in favor of pyyaml, so it should be easier to support in composes.